core/trie/trie.go

```diff
@@ -154,7 +173,9 @@ func (t *Trie) Get(key *felt.Felt) (*felt.Felt, error) {
 
 // Put updates the corresponding `value` for a `key`
 func (t *Trie) Put(key *felt.Felt, value *felt.Felt) error {
-	// Todo: check key is not bigger than max key value for a trie height.
+	if key.Cmp(t.maxKeyValue) != -1 {
+		return errors.New("key is bigger than the trie height")
+	}
```

Comment:
Shouldn't this check be added to `Get` as well? 

Also, the error message doesn't represent the error and is misleading.

```suggestion
	if key.Cmp(t.maxKeyValue) != -1 {
		return errors.New("key is bigger than the maxKeyValue")
	}
```

Reply:
In this scenario, the Get function would return "Key not found" so I am not sure if the check is required.